packages/blockchain-extension/extension/webview/TransactionView.ts

```diff
@@ -15,22 +15,23 @@
 'use strict';
 import * as vscode from 'vscode';
 import { ReactView } from './ReactView';
-import { ExtensionCommands } from '../../ExtensionCommands';
+import { TransactionCreateView } from './TransactionCreateView';
 
 export class TransactionView extends ReactView {
-    public appState: any;
+    protected context: any;
```

Comment:
You shouldn't need to do nor set `this.context = context`, as context is a property on View (which ReactView extends).
You can just call this.context with no problem!